ts/features/fims/singleSignOn/components/FimsErrorScreens.tsx

```diff
@@ -1,29 +1,59 @@
 import { useNavigation } from "@react-navigation/native";
 import { constNull } from "fp-ts/lib/function";
 import * as React from "react";
-import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import { useSelector } from "react-redux";
+import {
+  OperationResultScreenContent,
+  OperationResultScreenContentProps
+} from "../../../../components/screens/OperationResultScreenContent";
 import I18n from "../../../../i18n";
+import { fimsErrorStateSelector } from "../store/selectors";
+import { FIMS_SSO_ERROR_TAGS } from "../store/reducers";
 
-export const FimsMissingDataErrorScreen = () => {
+export const FimsSSOErrorScreen = ({
+  errorTag
+}: {
+  errorTag: FIMS_SSO_ERROR_TAGS;
```

Comment:
Is not it better if we define the props' type on another line and reference it here?

Reply:
since we are have a single prop, I found this the least cluttered way to handle this situation -- do you think it could improve readability?

Reply:
I think it should be kind of a convention that we should have, to always specify a non-inline type. Nonetheless, since the App team does not have such convention, let us leave it as it is